configyamls/all/otel-config.yaml

```diff
@@ -157,7 +157,7 @@ receivers:
       to: body
       type: move
   fluentforward:
-    endpoint: 0.0.0.0:8006
+    endpoint: 0.0.0.0:${MW_FLUENT_PORT}
```

Comment:
What if fluent port is passed as a command line argument instead of environment variable ?

Reply:
We are setting API Key and Target in otel-config.yaml in the same way.
I'll add something like this -- https://github.com/middleware-labs/mw-agent/blob/master/cmd/host-agent/main.go#L25 to handle the problem.

Reply:
Made the changes to fill the value even if values come from arguments.